### DISCUSSION TITLE
How to install

### DISCUSSION THREAD

**Discussion by DMikrut-iT** [netbox-community/devicetype-library#2386] [Q&A]
Hello,

I am a new NetBox user, browsing in github I came across the devicetype-library project and since I am new to these matters I do not know how to install this plugin on my own. Can someone tell me how to do this?

BR

**Comment by harryajc** [comment]
Hi @DMikrut-iT
Please look at the [Device-Type-Library-Import](https://github.com/netbox-community/Device-Type-Library-Import) repo which gives the steps to import data from this repo into your netbox instance.

Harry

**Reply by DMikrut-iT** [reply]
Thank you for your answer. Before writing I checked Device-Type-Library-Import](https://github.com/netbox-community/Device-Type-Library-Import), I followed the instructions step by step, unfortunately after entering my data I have this error
./nb-dt-import.py
Environment variable "REPO_URL" is not set.

My .env-test file looks like this:
NETBOX_URL=(netbox application www address)
NETBOX_TOKEN=(Netbox app token)
REPO_URL=https://github.com/netbox-community/devicetype-library.git
REPO_BRANCH=master
IGNORE_SSL_ERRORS=False
#SLUGS=c9300-48u isr4431 isr4331

**Comment by DMikrut-iT** [comment]
Thank you for your answer. Before writing I checked Device-Type-Library-Import](https://github.com/netbox-community/Device-Type-Library-Import), I followed the instructions step by step, unfortunately after entering my data I have this error
./nb-dt-import.py
Environment variable "REPO_URL" is not set.

My .env-test file looks like this:
NETBOX_URL=(netbox application www address)
NETBOX_TOKEN=(Netbox app token)
REPO_URL=https://github.com/netbox-community/devicetype-library.git
REPO_BRANCH=master
IGNORE_SSL_ERRORS=False
#SLUGS=c9300-48u isr4431 isr4331
